nethserver-subscription-inventory.spec

```diff
@@ -0,0 +1,41 @@
+%global debug_package %{nil}
+
+Summary: NethServer Subscriptions inventory agent
+Name: nethserver-subscription-inventory
+Version: 3.5.0
+Release: 1%{?dist}
+License: GPL
+URL: %{url_prefix}/nethserver-subscription
+Source0: %{name}-%{version}.tar.gz
+
+%ifarch x86_64
+Requires: puppet-agent
+%endif
+
+Provides: nethserver-inventory = %{version}
+Obsoletes: nethserver-inventory < %{version}
+
+%description
+NethServer Subscriptions inventory collects system facts and sends them every 
+day to a centralized server
+
+%prep
+%setup -q
+
+%build
+# noop
+
+%install
```

Comment:
either install the files only on x86_64 ie:
```
%install
%ifarch x86_64
install -m 0755 -D -T root/etc/cron.daily/nethserver-inventory %{buildroot}/etc/cron.daily/nethserver-i$
cp -av root/opt %{buildroot}/opt
(cd %{buildroot}; find . -type f | sed 's/^\.//' ) > %{name}-filelist
%endif
```
or ignore them in the `%files` section.

Otherwise we get (truncated)

```
error: Installed (but unpackaged) file(s) found:
   /etc/cron.daily/nethserver-inventory
   /opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/arp_macs.rb
   /opt/puppetlabs/puppet/lib/ruby/2.1.0/facter/backup.rb
...
...
```